airflow/stats.py

```diff
@@ -182,8 +182,8 @@ def stop(self, send=True):
             self.real_timer.stop()
 
 
-class DummyStatsLogger:
-    """If no StatsLogger is configured, DummyStatsLogger is used as a fallback."""
+class NullStatsLogger:
+    """If no StatsLogger is configured, NullStatsLogger is used as a fallback."""
```

Comment:
last time we had this naming dilemma empty won over null by great length

Reply:
Yeah, I remember that.  I don't know if "Empty" really applies in the case but I can't think of one that feels more accurate... in this case the object/class exists, it just doesn't DO anything...  If nobody comes up with a better-sounding answer, I'll change it to Empty I guess.